tck/src/test/java/org/eclipse/microprofile/jwt/tck/util/TokenUtilsGenerateTokenTest.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+ *
+ *  See the NOTICE file(s) distributed with this work for additional
+ *  information regarding copyright ownership.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.jwt.tck.util;
+
+import java.security.interfaces.RSAPublicKey;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+
+import org.eclipse.microprofile.jwt.Claims;
+import org.eclipse.microprofile.jwt.tck.TCKConstants;
+import org.jose4j.jwa.AlgorithmConstraints;
+import org.jose4j.jwt.JwtClaims;
+import org.jose4j.jwt.NumericDate;
+import org.jose4j.jwt.consumer.InvalidJwtException;
+import org.jose4j.jwt.consumer.JwtConsumerBuilder;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * Internal debugging tests
+ */
+public class TokenUtilsGenerateTokenTest {
+
```

Comment:
Within `DebugTest`, there was a `testHS256` test which doesn't seems to be retained. What is the reason for this?

Reply:
@rdebusscher yes, I was looking at it, it was just testing that Nimbus could use `HS256` without even using `TokenUtils`. And we won't support the symmetric signatures for 1.2. So I dropped it but as mentioned I've introduced an invalid `alg` test which actually stresses the relevant TokenUtils code